packages/jaeger-ui/src/components/TracePage/TracePageHeader/AltViewOptions.tsx

```diff
@@ -95,10 +98,11 @@ export default function AltViewOptions(props: Props) {
           >
             Trace JSON
           </Link>
-        </Menu.Item>
-      )}
-      {!disableJsonView && (
-        <Menu.Item>
+        ),
+        key: 'trace-json',
```

Comment:
nit: would be more readable if the key were the first field, since it's just one line. Writing it last makes it difficult to visually compare the indentation.

Reply:
Makes sense.